projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -437,6 +437,18 @@ export const isTrashcanItemSelected = (context: RuleContext): boolean => [naviga
  */
 export const canViewFile = (context: RuleContext): boolean => [hasFileSelected(context), navigation.isNotTrashcan(context)].every(Boolean);
 
+/**
+ * Checks if user can print the file.
+ * JSON ref: `canPrintFile`
+ *
+ * @param context Rule execution context
+ */
+export const canPrintFile = (context: RuleContext): boolean => {
```

Comment:
I think we have also some md documentation file/files related with rules? Can we add this there too?

Reply:
Added to rules.md documentation file